packages/experiments/src/components/Toggle/Toggle.state.ts

```diff
@@ -18,10 +20,17 @@ export class ToggleState extends BaseState<IToggleProps, IToggleViewProps, ITogg
       checked: !!props.defaultChecked,
       text: !!props.defaultChecked ? props.onText : props.offText,
       onChange: this._noop,
-      onClick: this._onClick
+      onClick: this._onClick,
+      toggleButton: this._toggleButton
     };
   }
 
+  public focus = () => {
+    if (this._toggleButton.current) {
+      this._toggleButton.current.focus();
+    }
+  };
```

Comment:
This seems a sensible addition but not as part of style variables.. did something trigger the need for this?

Reply:
In changing the component to use style variables I realized I had missed this as part of my previous PR where I transitioned Toggle to use Foundation.